android/cli/hooks/aar-transform.js

```diff
@@ -444,3 +445,102 @@ class SimpleFileCache {
 		fs.writeFileSync(this.cachePathAndFilename, dataToWrite);
 	}
 }
+
+/**
+ * Hyperloop versions below 2.2.0 have their own AAR handling which we need to
+ * disable by hooking into the affected hooks and reverting the changes Hyperloop
+ * made.
+ *
+ * @param {Object} cli CLI instance
+ * @param {Object} builder Builder instance
+ * @param {Object} appc Appc node utilities
+ * @param {Object} logger Logger instance
+ */
+function registerHyperloopCompatibilityFixes(cli, builder, appc, logger) {
+	var hyperloopModule = null;
+	builder.nativeLibModules.some(function (module) {
+		if (module.id === 'hyperloop' && appc.version.lt(module.version, '2.2.0')) {
+			hyperloopModule = module;
+			return true;
+		}
+	});
+	if (hyperloopModule === null) {
+		return;
+	}
+
+	var hyperloopBuildPath = path.join(builder.projectDir, 'build/hyperloop/android');
```

Comment:
does node handle unix slashes properly on Windows? I always tend to avoid using separators just in case:
`var hyperloopBuildPath = path.join(builder.projectDir, 'build', 'hyperloop', 'android');`